vignettes/getting-started-part-I.Rmd

```diff
@@ -0,0 +1,220 @@
+
+The package also provides a range of tools to convert data from line list, incidence, or other common formats into the required format (see [Data converters](https://package.epinowcast.org/dev/reference/index.html#data-converters)).
+
+```{r}
+latest_germany_hosp <- nat_germany_hosp |>
+  enw_latest_data() |>
+  enw_filter_reference_dates(remove_days = 40, include_days = 40)
+head(latest_germany_hosp, n = 10)
+```
+
+## Preprocessing
+
+Before modelling, the input data needs to be converted into the "reporting triangle" format (see [our model description for details](https://package.epinowcast.org/articles/model)). We also need to determine metadata to facilitate the model specification. This includes the number of days of data to use for the reference and report modules, the maximum delay to consider, and, optionally, a grouping (i.e. age group, location, or both) of observations.
+We process reported data into the format required for `epinowcast` and return it in a `data.table`. At this stage, we need to specify a grouping (i.e age, location) if any. 
+
+```{r}
+pobs <- enw_preprocess_data(retro_nat_germany, max_delay = 40)
+pobs
+```
+
+The returned output is in the form of a `data.table` with metadata stored as variables. It can be useful to check this output before specifying the model, just to make sure everything is as expected.
+
+# Model specification
+
+The `epinowcast` package is designed to provide users with a flexible and customizable modelling framework. The package comes equipped with several modules that users can utilize to construct models, and also allows users to create their own modules. These ensures that models can be tailored to the user's specific data and context.
+
+## Default nowcasting model
+
+The default nowcasting model in `epinowcast` consists of three modules: 
+
+- A process (expectation) module that models the expected counts by date of reference (`reference_date`)
+- A parametric reference reporting model which models the reporting delay distribution from the date of reference
+- A non-parametric reporting model which models differences in the reporting delay distribution by date of report (`report_date`), for example, day-of-the-week effects on the reporting delay.
+
+In the following sections, we specify simple models for each of these modules. The appropriateness of these specifications will vary depending on your context. See our vignettes for further details on model specification and examples of more complex models.
+
+## Process model
+
+A commonly used process model in nowcasting is to model the expected counts by date of reference via a geometric random walk as this acts as a minimally informed smoothing prior and thus gives a lot of weight to the observed data. This is the default process model in `epinowcast`. Users may also specify this model for themselves using the enw_expectation() function.
+
+```{r}
+expectation_module <- enw_expectation(
+  ~ 0 + (1 | day), data = pobs
```

Comment:
Because this is to "get started", I would normally prefer for the inputs to be assigned explicitly to the arguments. It's a nice way to introduce users to the argument names.

```suggestion
  r = ~ 0 + (1 | day), data = pobs
```

Reply:
seems reasonable to me; q to @seabbs - address this as part of improving the "getting started" guides generally or just do it now?

my general opinion for this amount / complexity: just do it here, but make a note of it as example of items to track down in detail as part of a PR dedicated to improving / expanding the guides.

Reply:
Yes I agree this really lies in improving the quick start. @jamesmbaazam what do you think? If in agreement would you be happy to make a new issue listing these issues as part of a wider effort to improve the quick start vignette?

Reply:
@jamesmbaazam if you're interested in taking this on as a more general improvement PR, recommend you create an issue for that, noting these classes of changes, then resolve these comments.

Reply:
Done.